DDDEastAnglia.Tests/SponsorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.Tests
+{
+    internal class SponsorBuilder
+    {
+        private byte[] logo;
+        private string name;
+        private DateTime? paymentDate;
+        private bool showPublicly = true;
+        private int sponsorId;
+        private int sponsorshipAmount;
+        private string url;
+
+        public Sponsor Build()
```

Comment:
I like the idea of the AtPlatinumLevel() idea.. very nice.

Justification:
For me a builders job is simple.. create a valid object every time.. with default values.. 

Example being is `new Sponsor()` is that valid? I don't want to have to this about it.. so I'll use a builder which will always create me a valid one.